### PR TITLE
Fix panpawn and Trickster & Add more Staff

### PR DESCRIPTION
**Pull request by Spandamn**

### CODE REVIEW

**Comment by Spandamn** [comment]
(Uncommented Tempanon since that'll allow testing his custom move)

File: mods/seasonal/moves.js [hunk 1]
```diff
@@ -518,9 +536,10 @@ exports.BattleMovedex = {
 	// panpawn
 	lafireblaze420: {
 		accuracy: 75,
-		basePower: 150,
+		basePower: 0,
+		damage: 150,
 		category: "Physical",
-		shortDesc: "20% chance to burn the target.",
+		shortDesc: "Does 150 damage. 20% chance to burn the target.",
```
Comment (HoeenCoder) [inline mods/seasonal/moves.js:562]: i thought this move was supposed to have 150 base power? not set damage

Comment (Spandamn) [reply mods/seasonal/moves.js:562]: In the bug reports issue, I read that it's 150 damage

Comment (CheeseMuffin) [reply mods/seasonal/moves.js:562]: I have asked panpawn and  he confirmed 150 damage

**Comment by Spandamn** [comment]
Since I was free today I also wrote for AuzBat

**Comment by Spandamn** [comment]
>I was JUST about to push a commit to add articblast
LMAO

**Comment by HoeenCoder** [comment]
rip

**Comment by Spandamn** [comment]
https://github.com/HoeenCoder/Pokemon-Showdown/commit/b91cddc285376bb834210270b45d931b88d6b826 well I gotta go so here's the commit

**Pull request merged**